bundle/manifests/storageos.com_storageosclusters.yaml

```diff
@@ -433,6 +433,33 @@ spec:
                           to an implementation-defined value. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/'
                         type: object
                     type: object
+                  snapshotControllerContainer:
+                    description: ResourceRequirements describes the compute resource
+                      requirements.
+                    properties:
+                      limits:
+                        additionalProperties:
+                          anyOf:
+                          - type: integer
+                          - type: string
+                          pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
```

Comment:
Where does this pattern come from? As I know these manifests are generated, so there must be a source for these patterns.

Reply:
Looks like it's defined here in controller-gen https://github.com/kubernetes-sigs/controller-tools/blob/a260f1391b0f722e07ae04fe87e23e32bc1a1ec7/pkg/crd/known_types.go#L69
Just a pattern to ensure resources are entered correctly

Reply:
Thanks a lot @nolancon it makes sense :)